context.go

```diff
@@ -11,10 +12,11 @@ var (
 
 var _ context.Context = &Context{}
 
-// A Context is the default routing context set on the root node of a
+// Context is the default routing context set on the root node of a
 // request context to track URL parameters and an optional routing path.
 type Context struct {
-	context.Context
+	// Parent context
+	parent context.Context
```

Comment:
@mrcpvn thanks for the note, however, chi v2 already breaks a lot of compatibility with v1, so that is okay. the chi.Context is itself a context.Context object, because it implements the context.Context interface, and uses the parent context.Context for behaviour (see implementations of the methods on chi.Context). 

@mrcpvn do you want to do this for testing..? we could export the "Parent", but I'd rather not unless we have to... this works https://github.com/pressly/chi/blob/master/mux_test.go#L833  .. is that not enough? let me know